languages/javalang/builder-renderer/java-builder-client/src/test/kotlin/io/vrap/codegen/languages/javalang/client/builder/BuilderTestCodeGenerator.kt

```diff
@@ -70,6 +74,15 @@ class BuilderTestCodeGenerator {
         generatorComponent.generateFiles()
     }
 
+    @Ignore
```

Comment:
Why did you add `@Ignore` here ?

Reply:
Honestly I dont even remember anymore, all of the tests in this class are ignored.

Reply:
There was some problem when running these tests.